zebra-utils/Cargo.toml

```
@@ -2,7 +2,7 @@
 name = "zebra-utils"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
 license = "MIT OR Apache-2.0"
-version = "1.0.0-alpha.1"
+version = "1.0.0-alpha.2"
```

zebra-utils hasn't been changed since the last zebra tag, so its version shouldn't be incremented.
```suggestion
version = "1.0.0-alpha.1"
```

---

Actually this is wrong, it depends on crates that have changed, so it needs an increment